test/spec/modules/lockrAIMIdSystem_spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+/* eslint-disable quotes */
+import * as lockrAIMSystem from "../../../modules/lockrAIMIdSystem.js";
+import { hook } from "../../../src/hook.js";
+import { expect } from "chai";
+import { coreStorage } from "../../../modules/userId/index.js";
+
+const defaultConfig = {
+  appID: "3b5a0f6c-7e91-11ec-b9c7-e330d98440a7",
+  email: "example@test.com",
+};
+
+const LIVE_RAMP_COOKIE = "_lr_env";
+const UID2_COOKIE = "_uid2_advertising_token";
+const ID5_COOKIE = "id5id";
+
```

Comment:
would you consider picking up sharedid (`_sharedid`) as well?

Reply:
A few points of clarity here: 

- lockr's AIM solely supports authenticated user events. Even with our integration with providers who can sometimes provide an pseudonymous identifier based on other attributes of the user agent, IP, etc. - we do not support that. Any and all data which is passed via lockr to an identity partner is solely hashed PII. As such, lockr AIM does not plan on facilitating SharedID as an option within its self-service UI at this time. Should this policy change, we will address this in the future. 

- lockr's AIM does provide pipes for publishers to pass data to various endpoints (S3 buckets, Optable, InfoSum, etc.). In these integrations we do include attributes the publisher might benefit from - i.e. a first party cookie as the root of the record. In this scenario we would be happy to pick up the SharedID as the key value, but this would require some engineering work on our end, which would have to be prioritized against other features publishers are requesting. At this point in time, this is not a function which has been requested. 

- To further clarify, lockr AIM does not interface as a bidder, so there is no need for "us" to pick up SharedID for normal programmatic.